ipinfo/utils_session.go

```diff
@@ -1,39 +1,9 @@
 package main
 
-import (
-	"io/ioutil"
-	"os"
-	"path/filepath"
-)
-
-func getTokenFilePath() (string, error) {
-	confDir, err := getConfigDir()
-	if err != nil {
-		return "", err
-	}
-
-	return filepath.Join(confDir, "token"), nil
-}
-
 func saveToken(tok string) error {
```

Comment:
Pretty much all of these functions can be deleted now, and we should just use the config functions and variables directly instead.